plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceUtils.java

```diff
@@ -206,6 +207,10 @@ public static DBPDataSourceContainer getDataSourceBySpec(
                         Map<String, String> handlerPopMap = handlerProps.computeIfAbsent(handlerId, k -> new HashMap<>());
                         handlerPopMap.put(paramName, paramValue);
                         handled = true;
+                    } else if (paramName.length() > PREFIX_AVANCED_PROP.length() && paramName.startsWith(PREFIX_AVANCED_PROP)) {
+                        paramName = paramName.substring(PREFIX_AVANCED_PROP.length());
+                        DBWorkbench.getPlatform().getPreferenceStore().setValue(paramName, paramValue);
```

Comment:
Well, it seems like it's not an "advanced prop" but rather a preference value. An application's preference, to be precise. Can we rename it?